receiver/awscontainerinsightreceiver/internal/host/ebsvolume.go

```diff
@@ -69,7 +72,14 @@ func newEBSVolume(ctx context.Context, session *session.Session, instanceID stri
 		osLstat:         os.Lstat,
 		evalSymLinks:    filepath.EvalSymlinks,
 	}
-
+	if configurer != nil {
+		err := configurer.Configure(awsmiddleware.SDKv1(&e.client.(*ec2.EC2).Handlers))
+		if err != nil {
+			log.Println("There was a problem configuring middleware on ec2 client")
+		} else {
+			log.Println("Successfully configured sdk with middleware handlers")
```

Comment:
Not sure if we need a log every time sdk is configured successfully. Similar comment for other places `Configure` is called